### ISSUE TITLE
An error happens in an unknown condition when opening a step dialog of Modified Java Script Value

### ISSUE THREAD

**Issue by HiromuHota** [HiromuHota/pentaho-kettle#43]
```
java.lang.IllegalArgumentException: Argument not valid
	at org.eclipse.swt.SWT.error(SWT.java:3794)
	at org.eclipse.swt.SWT.error(SWT.java:3731)
	at org.eclipse.swt.SWT.error(SWT.java:3702)
	at org.eclipse.swt.widgets.Item.setImage(Item.java:156)
	at org.eclipse.swt.widgets.MenuItem.setImage(MenuItem.java:278)
	at org.pentaho.di.ui.trans.steps.scriptvalues_mod.ScriptValuesModDialog.buildingFolderMenu(ScriptValuesModDialog.java:1892)
	at org.pentaho.di.ui.trans.steps.scriptvalues_mod.ScriptValuesModDialog.open(ScriptValuesModDialog.java:599)
	at org.pentaho.di.ui.spoon.delegates.SpoonStepsDelegate.editStep(SpoonStepsDelegate.java:127)
	at org.pentaho.di.ui.spoon.Spoon.editStep(Spoon.java:8796)
	at org.pentaho.di.ui.spoon.trans.TransGraph.editStep(TransGraph.java:3257)
	at org.pentaho.di.ui.spoon.trans.TransGraph.mouseDoubleClick(TransGraph.java:796)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:218)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:109)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:687)
	at org.eclipse.swt.widgets.Widget.notifyListeners(Widget.java:594)
	at org.eclipse.swt.widgets.Display.executeNextEvent(Display.java:1217)
	at org.eclipse.swt.widgets.Display.runPendingMessages(Display.java:1198)
	at org.eclipse.swt.widgets.Display.safeReadAndDispatch(Display.java:1181)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:1173)
	at org.eclipse.rap.rwt.application.AbstractEntryPoint.createUI(AbstractEntryPoint.java:69)
	at org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.createUI(RWTLifeCycle.java:177)
	at org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle$UIThreadController.run(RWTLifeCycle.java:290)
	at java.lang.Thread.run(Thread.java:745)
	at org.eclipse.rap.rwt.internal.lifecycle.UIThread.run(UIThread.java:107)
```

**Comment by HiromuHota** [comment]
Here is how to reproduce:
1. Opens webSpoon in a browser tab
2. Places the Modified Java Script Value step on the canvas and opens its dialog
3. Closes the tab
4. Opens webSpoon in another tab
5. Places the Modified Java Script Value step on the canvas and opens its dialog

**Comment by HiromuHota** [comment]
Resolved by 29767460551502a6a625adacbf93e2e78e219c79.
